### DISCUSSION TITLE
Do we know number of BlackArch users?

### DISCUSSION THREAD

**Discussion by MilindPurswani** [BlackArch/blackarch#4168] [Q&A]
Hi everyone, 

Has anyone successfully been able to identify how many BlackArch users do we have or maybe number of people who have downloaded BlackArch to date? 

Edit 1: Do we want us to be in a position where we are able to track this over time? Maybe not as fine grained as "who downloaded what how many times" but maybe something along the lines of "Number of downloads per month/per day/per year etc..."

I would really appreciate any help in this space!

Thanks

**Comment by anunna** [comment]
I have answered this previously. We do not track this and we also do not have control of the mirrors that are donated by the numerous organisations and volunteers.

**Reply by MilindPurswani** [reply]
Thanks @anunna, 

I updated the question a bit to add more color: 

I think the idea was to be able to put some numbers in quantifying number of downloads/number of users/number of applications or number of people consuming BlackArch or tools. How many folks end up downloading the latest update? etc...

I know we currently don't do it. But do we want ourselves to be in this state or are we open to bringing more light in this direction?

Thanks

**Comment by noraj** [comment]
For tracking packages installed we would need a [pkgstats](https://wiki.archlinux.org/title/pkgstats)-ba package installed by default.
